SPECS/openssl/openssl.spec

```diff
@@ -310,6 +310,8 @@ rm -f %{buildroot}%{_sysconfdir}/pki/tls/ct_log_list.cnf.dist
 %exclude %{_mandir}/man1*/openssl-tsget*
 
 %files libs
+%{!?_licensedir:%global license %%doc}
+%license LICENSE
```

Comment:
Thanks for fixing this! Since the `libs` subpackage seems to be required by all other ones, we could also remove `%license` from the default `%files` section. Not a requirement, though.

Reply:
I'll leave it, on the offchance anyone was relying on it.